paper-radio-button.html

```diff
@@ -147,29 +121,13 @@
       }
     },
 
-    onKeyDown: function(event) {
-      // Enter key.
-      if (event.keyCode === 13) {
-        this.onMouseDown();
-        event.preventDefault();
-      }
-    },
-
-    onMouseDown: function() {
-      if (this.disabled) {
-        return;
-      }
-
-      var old = this.checked;
-      this.checked = !this.isToggle || !this.checked;
-
-      if (this.checked !== old) {
-        this.fire('change');
-      }
+    _buttonStateChanged: function() {
```

Comment:
@sjmiles what do you think? I just based this on your initial implementation of `paper-toggle-button`

(going to merge this and let future monica deal with the answer in a different PR though)